urdf/package.xml

```diff
@@ -21,22 +21,29 @@
 
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
+  <build_depend>pluginlib</build_depend>
   <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
   <build_depend>urdfdom</build_depend>
   <!-- use ROS 2 package urdfdom_headers until upstream provides 1.0.0.-->
```

Comment:
Nit: this comment goes with `urdfdom_headers` one line below.

Reply:
Moved comment one line below in 6df48c9